core/src/main/java/com/redhat/lightblue/client/PropertiesLightblueClientConfiguration.java

```diff
@@ -57,6 +58,9 @@
     private static final String CERT_FILE_PATH_KEY = "certFilePath";
     private static final String CERT_PASSWORD_KEY = "certPassword";
     private static final String COMPRESSION = "compression";
+    private static final String READ_PREFERENCE = "readPreference";
+    private static final String WRITE_CONCERN = "writeConcern";
+    private static final String MAX_QUERY_TIME_MS = "maxQueryTimeMS";
 
```

Comment:
I changed these to be "execution.mongo.*" to help distinguish if we do add execution options for another backend.